sigma9.css

```diff
@@ -237,8 +244,8 @@ sup {
 	max-height: 0;
 	color: #eee;
 	background: transparent;
-	font-family: BauhausLTDemi, 'Nanum Gothic', Arial, sans-serif;
-	font-size: 180%;
+	font-family: 'Sans Normalcy', 'Nanum Gothic', Arial, sans-serif;
+	font-size: 150%;
```

Comment:
We should change all the % font-sizes to em sizes instead. So, in this case, it'd be `1.5em`.